src/Config/user_local.props

```diff
@@ -3,9 +3,9 @@
   <PropertyGroup>
     <RevitVersionNumber Condition=" '$(RevitVersionNumber)' == '' ">2017</RevitVersionNumber>
 	<REVIT_VERSION>Revit_$(RevitVersionNumber)</REVIT_VERSION>
-	<DYNAMOTESTAPI Condition=" '$(DYNAMOTESTAPI)' == '' ">$(SolutionDir)..\..\Dynamo\bin\$(Platform)\$(Configuration)</DYNAMOTESTAPI>
+	<DYNAMOTESTAPI Condition=" '$(DYNAMOTESTAPI)' == '' ">C:\Program Files\Dynamo\Dynamo Core\1.2</DYNAMOTESTAPI>
```

Comment:
this is not correct, there should be no changes to these csproj files at all, you can reset them using git to the previous commit.

then to make sure the build works, pull Dynamo master and build that first.